src/vs/workbench/contrib/chat/browser/chatContentParts/media/chatConfirmationWidget.css

```diff
@@ -12,6 +12,10 @@
 	align-items: center;
 }
 
+.chat-confirmation-widget .monaco-text-button {
+	height: 28px;
```

Comment:
I recommend not hard-coding heights like this, especially since chat font props are now customizable, what if I set a font-size larger than this?

Instead, figure out what is constraining the width and causing it to wrap